post_installation_configuration/cluster-tasks.adoc

```diff
@@ -544,21 +544,6 @@ include::modules/machineset-delete-policy.adoc[leveloffset=+2]
 
 include::modules/nodes-scheduler-node-selectors-cluster.adoc[leveloffset=+2]
 
-include::modules/installation-extend-edge-nodes-aws-local-zones.adoc[leveloffset=+2]
-
-.Next steps
-
-* Optional: Use the AWS Load Balancer (ALB) Operator to expose a pod from a targeted edge worker node to services that run inside a Local Zone subnet from a public network. See xref:../networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc#nw-installing-aws-load-balancer-operator_aws-load-balancer-operator[Installing the AWS Load Balancer Operator].
```

Comment:
This removes the AWS work from the Post-installation configuration / Cluster tasks section. See https://docs.openshift.com/container-platform/4.13/post_installation_configuration/cluster-tasks.html#installation-extend-edge-nodes-aws-local-zones_post-install-cluster-tasks